lib/src/timing.dart

```diff
@@ -19,12 +26,20 @@ class TimeSlice {
 
   TimeSlice(this.startTime, this.stopTime);
 
+  /// Constructs TimeSlice from JSON representation
+  factory TimeSlice.fromJson(Map<String, dynamic> json) =>
+      _$TimeSliceFromJson(json);
+
+  /// Converts to JSON representation
```

Comment:
[nit] I don't think this comment is necessary. The comment on the factory constructor probably isn't necessary either. They aren't saying anything the signature doesn't already say.